chromium_src/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.cc

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2019 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "base/callback.h"
+#include "brave/browser/browsing_data/brave_browsing_data_remover_delegate.h"
+#include "brave/components/brave_rewards/browser/rewards_service.h"
+#include "brave/components/brave_rewards/browser/rewards_service_factory.h"
+#include "chrome/browser/profiles/profile.h"
+
+#include "../../../../chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.cc"  // NOLINT
+
+using brave_rewards::RewardsService;
+using brave_rewards::RewardsServiceFactory;
+
+void ChromeBrowsingDataRemoverDelegate::RemoveRewardsEmbedderData(
+    int remove_mask,
+    base::OnceClosure callback,
+    Profile* profile_) {
+
+  //  AUTO-CONTRIBUTE DATA
+  if (remove_mask &
+    BraveBrowsingDataRemoverDelegate::DATA_TYPE_REWARDS_AUTO_CONTRIBUTE) {
+    RewardsService* rewards_service_ =
+      RewardsServiceFactory::GetForProfile(profile_);
+    if (rewards_service_) {
+      rewards_service_->RemoveAutoContributeData(
+      base::Bind(
+        &ChromeBrowsingDataRemoverDelegate::OnRemoveAutoContributeData,
+        weak_ptr_factory_.GetWeakPtr()));
+    }
+  }
+
+  //  ALL REWARDS DATA
+  if (remove_mask &
+    BraveBrowsingDataRemoverDelegate::DATA_TYPE_REWARDS_ALL_DATA) {
+    RewardsService* rewards_service_ =
+      RewardsServiceFactory::GetForProfile(profile_);
+    if (rewards_service_) {
+      rewards_service_->RemoveAllRewardsData(
+      base::Bind(
+        &ChromeBrowsingDataRemoverDelegate::OnRemoveAllRewardsData,
+        weak_ptr_factory_.GetWeakPtr()));
+    }
+  }
+}
+
+void ChromeBrowsingDataRemoverDelegate::OnRemoveAutoContributeData(
+    bool result) {
+  if (!result) {
+    // log error
```

Comment:
Should this be a `// TODO:` or were you planning on adding this before merging?

Reply:
Will be address

Reply:
This function was removed.